storage_service/locations/signals.py

```diff
@@ -63,30 +63,27 @@ def _log_report(uuid, success, message=None):
     from . import models
 
     package = models.Package.objects.get(uuid=uuid)
-    models.FixityLog.objects.create(
+    return models.FixityLog.objects.create(
         package=package, success=success, error_details=message
     )
 
 
 @receiver(failed_fixity_check, dispatch_uid="fixity_check")
 def report_failed_fixity_check(sender, **kwargs):
     report_data = json.loads(kwargs["report"])
-    _log_report(kwargs["uuid"], False, report_data["message"])
+    fixity_log = _log_report(kwargs["uuid"], False, report_data["message"])
+    timestamp = fixity_log.datetime_reported.strftime("%Y-%m-%d %H:%M:%S")
```

Comment:
Correct me if I'm wrong, but I think this needs to be a [localtime](https://docs.djangoproject.com/en/4.2/ref/utils/#django.utils.timezone.localtime)?

It's 16:47 here right now and I just set up your branch in the development environment which by default uses `America/Los_Angeles` as the `TIME_ZONE` and when I call the check fixity endpoint I get a timestamp that it's in the future:

```console
archivematica-storage-service-1  | [2024-05-17 22:47:32] A fixity check failed for the package with UUID 8c9c347e-7fd4-442e-8a6f-b02338ba2b4e.
```